app/input/Input.tsx

```diff
@@ -0,0 +1,12 @@
+import type { PropsWithChildren } from "react";
+import {
+	Input as ChakraInput,
+	InputProps,
+	useColorModeValue,
+} from "@chakra-ui/react";
+
+export default function Input(props: PropsWithChildren<InputProps>) {
+	const focusBorderColor = useColorModeValue("brand.500", "brand.300");
+
+	return <ChakraInput {...props} focusBorderColor={focusBorderColor} />;
```

Comment:
This seems oddly specific... why would we change the default `focusBorderColor`?